pages/index/_learn-more.html

```diff
@@ -27,10 +27,10 @@ <h2>Get access to the Infrastructure as Code Library.</h2>
     <img src="/assets/img/infrastructure-cube-icons/infrastructure-design-your-stack-grunty.png" />
   </div>
   <div class="col-xs-12 col-sm-7 col-sm-pull-5 col-md-6 col-md-pull-6 col-description">
-    <h2>Deploy your entire tech stack.</h2>
+    <h2>Deploy your entire tech stack.</h2> <!-- PR Review Callout: Should this header change given its content vs the section above? -->
```

Comment:
Not sure I understand the question?

Reply:
After trying to re-state my question, I realize it is not really a concern. I will remove the comment.